app/containers/Scatter/saga.js

```diff
@@ -55,6 +56,31 @@ function* watchScatterConnect() {
   yield takeLatest(CONNECT_ACCOUNT, getEosAccount);
 }
 
+//
+// Refresh account data
+//
+
+function* refreshEosAccountData() {
+  const accountName = yield select(makeSelectEosAccount());
+  const eosClient = yield select(EosClient());
+  try {
+    if (accountName && accountName !== 'Attach an Account') {
+      const account = yield eosClient.getAccount(accountName);
+      const currency = yield eosClient.getCurrencyBalance('eosio.token', accountName);
+      account.currency = currency;
+      yield put(refreshedAccountData(account));
+    } else {
+      yield put(refreshedAccountData(null));
+    }
+  } catch (err) {
+    // console.log(err);
```

Comment:
Shouldn't we be using Notifications here?

Reply:
No I want this to happen in the background. We could use its own set of loading state though if we want spinners.

Reply:
Right gotcha! Should go to Kibana or something someday then